core/config/application_config.go

```diff
@@ -20,6 +20,7 @@ type ApplicationConfig struct {
 	ImageDir                            string
 	AudioDir                            string
 	UploadDir                           string
+	ConfigsDir                          string
```

Comment:
it maybe makes more sense to call this FilesDir, at a first glance I thought this was a directory for LocalAI config files